playbooks/bind_config.yaml

```
@@ -0,0 +1,32 @@
+---
+- name: Manage Bind9 configurations
+  hosts: bind_servers
+  become: true
+
+  tasks:
+    - name: Synchronize Bind9 configurations
+      ansible.builtin.copy:
+        src: "{{ item }}"
+        dest: /etc/bind/
```

```suggestion
        dest: /etc/bind
```

(Completely optional. For consistency with the rest of the codebase.)